### PR TITLE
Add test for argument with empty prefix and separate false

### PR DESCRIPTION
**Pull request by bogdang989**
Test for https://github.com/common-workflow-language/cwltool/issues/1165

### CODE REVIEW

File: conformance_tests.yaml [hunk 1]
```diff
@@ -3313,3 +3313,13 @@
     Test that optional number output is returned as 0, not null
   tags: [ required, inline_javascript, command_line_tool ]
   id: 253
+
+- tool: tests/arg-empty-prefix-separate-false.cwl
+  job: tests/empty.json
+  output: {}
```
Comment (mr-c) [inline conformance_tests.yaml]: What is the expected output?

**Review by mr-c — CHANGES_REQUESTED** [review]

File: conformance_tests.yaml (hunk 1 above)
Comment (bogdang989) [reply conformance_tests.yaml]: The idea was to run command `echo non_existing_app` if everything works fine, and run `non_existing_app` if executor has a bug, so basically just relying on exit code as a check. Added a redirect to stdout so the output size and checksum are also validated.

**Comment by mr-c** [comment]
Since this is a change in behaviour, I guess it would have to wait for CWL v1.3 or 2.0...
